sql/item_weapon.sql

```diff
@@ -4371,20 +4385,42 @@ INSERT INTO `item_weapon` VALUES (21496,'furious_broth',0,0,0,0,0,0,1,2215,3154,
 INSERT INTO `item_weapon` VALUES (21497,'rapid_broth',0,0,0,0,0,0,1,2216,5203,0);
 INSERT INTO `item_weapon` VALUES (21498,'crackling_broth',0,0,0,0,0,0,1,2217,3412,0);
 INSERT INTO `item_weapon` VALUES (21499,'creepy_broth',0,0,0,0,0,0,1,2218,3669,0);
-INSERT INTO `item_weapon` VALUES (21504,'arasy_sainti',1,0,242,242,188,0,1,531,96,0);
-INSERT INTO `item_weapon` VALUES (21505,'arasy_sainti_+1',1,0,242,242,188,0,1,529,97,0);
-INSERT INTO `item_weapon` VALUES (21506,'jolt_counter',1,0,242,242,188,0,1,576,148,0);
-INSERT INTO `item_weapon` VALUES (21507,'jolt_counter_+1',1,0,242,242,188,0,1,571,149,0);
+INSERT INTO `item_weapon` VALUES (21504,'arasy_sainti',1,0,242,242,188,0,1,561,110,0);
+INSERT INTO `item_weapon` VALUES (21505,'arasy_sainti_+1',1,0,242,242,188,0,1,559,111,0);
+INSERT INTO `item_weapon` VALUES (21506,'jolt_counter',1,0,242,242,188,0,1,566,183,0);
+INSERT INTO `item_weapon` VALUES (21507,'jolt_counter_+1',1,0,242,242,188,0,1,564,184,0);
+INSERT INTO `item_weapon` VALUES (21511,'hep._baghnakhs',1,0,0,0,0,0,1,570,161,0);
+INSERT INTO `item_weapon` VALUES (21512,'hep._baghnakhs_+1',1,0,0,0,0,0,1,561,162,0);
+INSERT INTO `item_weapon` VALUES (21513,'raetic_baghnakhs',1,0,0,0,0,0,1,531,163,0);
+INSERT INTO `item_weapon` VALUES (21514,'rae._baghnakhs_+1',1,0,0,0,0,0,1,546,164,0);
```

Comment:
delay 561
ilvl_skill 242
ilvl_macc 215

if ilvl_parry is used for guard, ilvl_parry also needs to be 242, if not, ilvl_guard needs to be created, and ilvl_guard 242.